### DISCUSSION TITLE
Mobile app - do not show photos on internal storage

### DISCUSSION THREAD

**Discussion by jdumeez** [immich-app/immich#4990] [Feature Request]
### The feature

Dear, 

i have a workflow where all photos (own phone/DLSR/wife phone) get consolidated on an SMB share. 
This SMB share is linked to immich as external library. This consolidates our pictures from different devices and works amazing on the web client. 

However, on the mobile client, this workflow duplicates the photos that were taken with the phone itself. 
As the photo on the external library, and the storage on phone contain the same photos. 

So i have a feature request to have a toggle to disable showing the photos from the internal storage of the phone in the gallery of the mobile app. 

Keep up the good work,
thx! 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Luvbe3** [comment]
[Feature] add a button to show/hide local photos in the app #5092  

it's hard to differentiate between photos on the phone and on the cloud immich. sometimes i want to just look at the photos on the cloud but i keep getting distracted by the photos on the phone. it would be great if this feature was implemented!

**Comment by archont00** [comment]
Excluding user defined local folders from Immich processing would be a more universal option.

**Reply by alextran1502** [reply]
You can just not select any local albums for backup, then the instance will only show the remote media
